docs/user-guide/creating-application/deployment-template/deployment.md

```diff
@@ -12,22 +9,168 @@ This is the default deployment chart. You can select `Deployment` chart when you
+
 | Key | Descriptions |
 | :--- | :--- |
-| `Chart version` | Select the Chart Version using which you want to deploy the application.<br> Refer [Chart Version](../../creating-application/deployment-template/rollout-deployment.md#1.-chart-version) section for more detail.</br> |
-| `Basic Configuration` | You can select the basic deployment configuration for your application on the **Basic** GUI section instead of configuring the YAML file.<br>Refer [Basic Configuration](../../creating-application/deployment-template/rollout-deployment.md#2.-basic-configuration) section for more detail.</br>|
-| `Advanced (YAML)` | If you want to do additional configurations, then click **Advanced (YAML)** for modifications.<br>Refer [Advanced (YAML)](../../creating-application/deployment-template/rollout-deployment.md#3.-advanced-yaml) section for more detail.</br> |
-| `Show application metrics` | You can enable `Show application metrics` to see your application's metrics-CPU Service Monitor usage, Memory Usage, Status, Throughput and Latency.<br>Refer [Application Metrics](../../creating-application/app-metrics.md) for more detail.</br> |
+| `Chart Version` | Select the Chart Version using which you want to deploy the application. |
+
+Devtron uses helm charts for the deployments. And we are having multiple chart versions based on features it is supporting with every chart version.
+
+One can see multiple chart version options available in the drop-down. you can select any chart version as per your requirements. By default, the latest version of the helm chart is selected in the chart version option.
+
+Every chart version has its own YAML file. Helm charts are used to provide specifications for your application. To make it easy to use, we have created templates for the YAML file and have added some variables inside the YAML. You can provide or change the values of these variables as per your requirement.
+
+If you want to see [Application Metrics](#4.-show-application-metrics) (as an example, Status codes 2xx, 3xx, 5xx; throughput, and latency etc.) for your application, then you need to select the latest chart version.
+
+**Note**: Application Metrics are not supported for the Chart version older than 3.7 version.
+
+---
+
+## 2. Basic Configuration
+
+Some of the use-cases which are defined on the Deployment Template (YAML file) may not be applicable to configure for your application. In such cases, you can do a basic deployment configuration for your application in the **Basic (GUI)** section instead of configuring the YAML file.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/deployment-template/basic-config-deployment-template-v2.jpg)
+
+The following fields are available for you to edit in the **Basic (GUI)** section:
+
+| Fields | Description |
+| :---    |     :---       |
+| **Arguments**  | Enable the `Arguments` to pass one or more argument values. By default, it is in the `disabled` state. |
```

Comment:
Stuff below this is common for all devtron charts except cron job (It may have some fields lesser), why mentioning this in deployment and not in common? Same is true for API def to modify the schema, defined below

Reply:
Correct. Fixed. Now I have put all those in common (Base Deployment Template doc).
Originally, things were stuffed only in Rollout Deployment doc (because that used to be the default type long ago, I guess).